### PR TITLE
add package server upgrade steps

### PR DESCRIPTION
**Pull request by bbodenmiller**
Add missing package server upgrade details. Addresses comments in #25.

### CODE REVIEW

File: doc/update/README.md [hunk 1]
```diff
@@ -13,10 +13,12 @@ configured the packages to run this command automatically after the new package
 is installed. If you are installing GitLab 7.9 or earlier, please see the
 [procedure below](#updating-from-gitlab-66-and-higher-to-the-latest-version).
 
-First, download the latest [CE](https://packages.gitlab.com/gitlab/gitlab-ce) or
+If you installed using the package server all you need to do is run `sudo apt-get update && sudo apt-get install gitlab-ce` (for Debian/Ubuntu) or `sudo yum install gitlab-ce` (for CentOS/Enterprise Linux).
```
Comment (axilleas) [inline doc/update/README.md]: `package server` seems confusing. What about 

```
If you installed GitLab using the [official repository](https://packages.gitlab.com/gitlab) all you need...
```

Comment (bbodenmiller) [inline doc/update/README.md]: I'm fine with whatever, just copied from https://about.gitlab.com/update/. Seems to be a consistency issue around the proper term.

Comment (axilleas) [inline doc/update/README.md]: Oh yeah, I hadn't payed attention what the site said. Better be consistent, don't change anything for the time being. Thanks

Comment (bbodenmiller) [inline doc/update/README.md]: Even website is inconsistent: `package server` and `repo server`

Comment (bbodenmiller) [inline doc/update/README.md]: Fixed consistency issue in MR at https://gitlab.com/gitlab-com/www-gitlab-com/merge_requests/877.

**Comment by maxlazio** [comment]
Thanks @bbodenmiller

**Pull request merged**